pkg/services/serviceaccounts/errors.go

```diff
@@ -0,0 +1,14 @@
+package serviceaccounts
+
+import (
+	"fmt"
+)
+
+type ErrServiceAccountNotFound struct {
+	ID    int64
+	OrgID int64
+}
+
+func (e ErrServiceAccountNotFound) Error() string {
+	return fmt.Sprintf("Service account with id %d for orgID %d not found", e.ID, e.OrgID)
+}
```

Comment:
What purpose does this enable?  The caller already knows what the ID and orgID were, because they provided it.  A standard error does the same job.